shuup/reporting/reports/sales.py

```diff
@@ -0,0 +1,126 @@
+# -- encoding: UTF-8 --
+import itertools
+
+import six
+from django.db.models import Sum
+from django.utils.translation import ugettext_lazy as _
+
+from shuup.core.models import Order
+from shuup.core.pricing import TaxfulPrice, TaxlessPrice
+from shuup.reporting.reporter.base import ShuupReportBase
+
+
+class OrderReportMixin(object):
+    def get_objects(self):
+        return Order.objects.filter(
+            shop=self.shop, order_date__range=(self.start_date, self.end_date)).order_by("order_date")
+
+
+class SalesReport(OrderReportMixin, ShuupReportBase):
```

Comment:
I would like to separate these reports a little maybe even one report per file under reports or reports/sales? I can easily see this file go out of hand.